query_optimizer/ExecutionGenerator.cpp

```diff
@@ -1030,9 +1184,20 @@ void ExecutionGenerator::convertCreateTable(
     catalog_relation->setDefaultStorageBlockLayout(layout.release());
   }
 
+  if (physical_plan->partition_scheme()) {
+    LOG(INFO) << "Relation created with partition scheme." << std::endl;
+    std::size_t num_partitions = physical_plan->partition_scheme()->getPartitionSchemeHeader().getNumPartitions();
+    attribute_id part_attr_id = physical_plan->partition_scheme()->getPartitionSchemeHeader().getPartitionAttributeId();
+    PartitionSchemeHeader *part_scheme_header = new HashPartitionSchemeHeader(num_partitions, part_attr_id);
```

Comment:
I was wondering if we have a particular reason to use hash-based partition by default, even if the table to create has specify its partition scheme (i.e., range-based).